src/compiler/binder.ts

```diff
@@ -3495,6 +3495,9 @@ namespace ts {
             case SyntaxKind.BreakStatement:
                 transformFlags |= TransformFlags.ContainsHoistedDeclarationOrCompletion;
                 break;
+
+            case SyntaxKind.ImportCallExpression:

```

Comment:
Just a note: there were talks in the past about attaching meta-properties to the `import` keyword (similar to `new.target`), though there hasn't been a concrete proposal for it yet. It'll probably become needed as node iterates on their implementation.

Also, estree and babylon went with `Import` for this AST node.